parser.go

```diff
@@ -25,7 +25,8 @@ import (
 	"unicode"
 )
 
-var pythonMultiline = regexp.MustCompile("^(\\s+)([^\n]+)")
+const minReaderBufferSize = 64*1024
+var pythonMultiline = regexp.MustCompile(`^([\t\f ]+)(.*)`)
```

Comment:
Would you mind explain what has changed in this regex? (I'm a rookie about regex)

Reply:
\s is equivalent to [\r\n\t\f\v ], e.g. not includes newlines and carriage returns. that is a problem when trying to parse a multiline value one line at a time. It matched empty lines without indents which made them part of the value when they shouldn't. Multiline values are terminated when a line is encountered that has a lesser whitespace indent than the first line after the line with the key. By using \s it was not possible to catch the termination condition correctly.

Reply:
Thanks for the explanation!